user-guide/Advanced_Functionality/Databases/Supported_system_data_storage_architectures/Dedicated_clustered_storage.md

```diff
@@ -74,13 +74,11 @@ While hosting Elasticsearch and DataMiner nodes on the same machine is already n
 
 ## Failover setups (with geo-redundancy)
 
-To achieve geo-redundancy and reduce latency between DMAs deployed across the globe, typically data center setups are used. In case you need a setup with multiple data centers deployed worldwide, please contact Skyline.
+To achieve geo-redundancy and reduce latency between DMAs deployed across the globe, typically data center setups are used. In case you need a setup with multiple data centers deployed worldwide, please contact Skyline. For geo-redundant systems we require a **stable network connection between the sites that has low latency**. Setting up an OpenSearch/Elasticsearch cluster across high-latency nodes is not advised, as this could compromise proper functionality. This means that it is not possible to simply take OpenSearch/Elasticsearch nodes and spread them out over two or more locations if there is high latency between those locations. This restriction specifically applies to latency between the openSearch/Elasticsearch nodes, not between the DataMiner nodes and the OpenSearch/Elasticsearch nodes.
 
-Failover setups with Amazon Keyspaces Service, Azure Managed Instance for Apache Cassandra Service, and Amazon OpenSearch Service are considered deprecated as of DataMiner 10.3.0 [CU9]/10.3.12.
+Failover setups with Amazon Keyspaces Service, Azure Managed Instance for Apache Cassandra Service, and Amazon OpenSearch Service are considered deprecated as of DataMiner 10.3.0 [CU9]/10.3.12. If you want to use cloud storage instead, we recommend switching to [Storage as a Service (STaaS)](xref:STaaS) in which we have geo-redundant options.
 
-Note that **setting up an OpenSearch/Elasticsearch cluster across high-latency nodes is not advised**, as this could compromise proper functionality. This means that it is not possible to simply take OpenSearch/Elasticsearch nodes and spread them out over two or more locations if there is high latency between those locations. This restriction specifically applies to latency between the openSearch/Elasticsearch nodes, not between the DataMiner nodes and the OpenSearch/Elasticsearch nodes. This is why we recommend that instead data is offloaded to multiple OpenSearch clusters.
+The figure below illustrates the recommended minimum setup for geo-redundancy with on-premises machines. For Cassandra, the built-in [NetworkTopologyStrategy](https://cassandra.apache.org/doc/4.0/cassandra/cql/ddl.html#networktopologystrategy) is used in order to configure geo-redundancy. For OpenSearch, we configure one cluster for which the nodes are spread over two data centers and there one tie-breaker in a third site (can be hosted in the cloud).
```

Comment:
for which the nodes are spread over three zones, two of which host the data whilst the third zone has a single node which does not handle any data and functions as tie-breaker for cluster configuration. The tie-breaker node can be cloud hosted if the latency is sufficiently low (sub 50 ms). For this we leverage [shard allocation awareness](https://opensearch.org/docs/2.12/tuning-your-cluster/#advanced-step-6-configure-shard-allocation-awareness-or-forced-awareness).